docker/mysql/mysql.cnf

```diff
@@ -8,6 +8,13 @@ default-character-set = utf8mb4
 skip-host-cache
 skip-name-resolve
 
+sort_buffer_size = 16M
+innodb_buffer_pool_size = 2G
+innodb_log_buffer_size = 512M
+innodb_log_file_size = 1G
+innodb_write_io_threads = 8
+innodb_flush_log_at_trx_commit = 0
```

Comment:
For contrast, these are the default settings:
```
sort_buffer_size = 2M
innodb_buffer_pool_size = 128M
innodb_log_buffer_size = 16M
innodb_log_file_size = 100M
innodb_write_io_threads = 4
innodb_flush_log_at_trx_commit =1
```

Reply:
I'll test this a bit later - I had issues with `sort_buffer_size` being too high with a less powerful docker setup.